src/Home/Home.spec.tsx

```diff
@@ -1,101 +1,255 @@
 import React from "react"
-import { Alert, Platform } from "react-native"
-import { render, cleanup, wait, fireEvent } from "@testing-library/react-native"
+import { Alert } from "react-native"
+import { fireEvent, render, wait, within } from "@testing-library/react-native"
+import { useNavigation } from "@react-navigation/native"
+import { useSafeAreaInsets } from "react-native-safe-area-context"
 import "@testing-library/jest-native/extend-expect"
 
-import { ENPermissionStatus } from "../PermissionsContext"
 import Home from "./Home"
+import { PermissionsContext, ENPermissionStatus } from "../PermissionsContext"
+import { PermissionStatus } from "../permissionStatus"
+import { isBluetoothEnabled } from "../gaen/nativeModule"
+import { isPlatformiOS } from "../utils/index"
 
-afterEach(cleanup)
+jest.mock("@react-navigation/native")
```

Comment:
Some of this mocking was being shielded by having a screen component and a "launcher" component. I agree that this one is testing the native API more closely, but it also makes these tests way harder to reason and definitely harder to mock. Solid work here getting it to pass 👏